en/developer_guide/install_with_vagrant_for_contributors.rst

```diff
@@ -0,0 +1,211 @@
+
+.. code-block:: bash
+
+    $ php -r "readfile('https://getcomposer.org/installer');" | php
+
+see `Download Composer`_
+
+Install OpenOrchestra
+---------------------
+To contribute on the project you have to install two versions: the master version and the last stable one.
+Here is the directory tree with your two projects that we recommand:
+
+.. code-block::
+
+    |_ install-directory
+          |_ open-orchestra-master
+                |_ open-orchestra
+                |_ open-orchestra-front-demo
+                |_ open-orchestra-media-demo
+                |_ open-orchestra-provision
+          |_ open-orchestra-stable
+                |_ open-orchestra
+                |_ open-orchestra-front-demo
+                |_ open-orchestra-media-demo
+                |_ open-orchestra-provision
+
+
+- To install the master open-orchestra parts using ``composer``:
+
+  In your ``open-orchestra-master`` directory:
+  
+  .. code-block:: bash
+
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra ./open-orchestra -s dev --ignore-platform-reqs --no-scripts --keep-vcs dev-master
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-front-demo ./open-orchestra-front-demo -s dev --ignore-platform-reqs --no-scripts --keep-vcs dev-master
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-media-demo ./open-orchestra-media-demo -s dev --ignore-platform-reqs --no-scripts --keep-vcs dev-master
+
+  Clone the provisioning repository :
+
+  .. code-block:: bash
+
+    $ git clone git@github.com:open-orchestra/open-orchestra-provision.git
+
+
+- To install the stable open-orchestra parts using ``composer``:
+
+  In your ``open-orchestra-stable`` directory:
+  
+  .. code-block:: bash
+
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra ./open-orchestra -s stable --ignore-platform-reqs --no-scripts --keep-vcs
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-front-demo ./open-orchestra-front-demo -s stable --ignore-platform-reqs --no-scripts --keep-vcs
+    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-media-demo ./open-orchestra-media-demo -s stable --ignore-platform-reqs --no-scripts --keep-vcs
+
+  Clone the stable provisioning repository. Don’t forget to specify the last stable version branch with the ``--branch`` option.
+
+  .. code-block:: bash
+
+    $ git clone git@github.com:open-orchestra/open-orchestra-provision.git --branch=1.0
+
+Override the dns redirection
+----------------------------
+
+In the ``/etc/hosts`` file of your computer add the following lines :
+
+.. code-block:: text
+
+    192.168.33.10   admin.openorchestra.dev
+    192.168.33.10   demo.openorchestra.dev
+    192.168.33.10   media.openorchestra.dev
+
+The two installs use the same IP addresses, so you can’t launch both vagrant at the same time.
```

Comment:
that's misleading, the IP configuration should probably be explained so people can use both VMs at the same time